docs/virtual-networking-setup.rst

```diff
@@ -21,24 +21,23 @@ can still create your own. xenbr0 is created in most versions of RHEL by xend st
 Networking
 ##########
 
-Virtualization networking in koan uses "bridged" mode. This is so that guests by default can be connected to from the
+Virtualization networking in Koan uses "bridged" mode. This is so that guests by default can be connected to from the
 outside world, which is very important for them to be able to do useful things.
 
-If a network bridge already exists, koan will be able to use that, though in some cases, you'll have to create your own
-bridge in order to get koan to work. You do this by modifying the network configuration on your virtual host, and then
-using the koan parameter ``--virt-bridge=bridgename``.
+If a network bridge already exists, Koan will be able to use that, though in some cases, you'll have to create your own
+bridge in order to get Koan to work. You do this by modifying the network configuration on your virtual host, and then
+using the Koan parameter ``--virt-bridge=bridgename``.
 
 (As we mentioned, if you use virbr0, it's a fake bridge, so be aware you won't be able to ssh into your guests...
 However koan can use that if you REALLY want to)
```

Comment:
When you capitalize koan everywhere, you need to do it here as well

Reply:
Thanks! It seems that I missed some of them. Hopefully this is fixed now.